src/api/enums/HashableObjectType.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * HashableObjectTypeEnum
+ *
+ */
+
+export enum HashableObjectType {
+
+    LISTINGITEMMESSAGE = 'listingItemMessage'
+
```

Comment:
we dont need hashes for Messages, they are just representations of post ListingItemsTemplates and for them a hash should be created before its posted as Message.
so hashableobjecttypes should at least include ListingItemTemplate, ListingItem (pretty much the same thing but anyway) and ItemImage